metabase/metabase
# Add semantic type icons to metadata editor

This PR adds semantic type icons to the data model metadata editor

### To Verify

1. Open a data model
2. Click the data model name to open the details sidebar
3. Click the "Customize metadata" button
4. Make sure every column with a specified special (semantic) type has an icon other than "ellipsis" (three dots)
5. Make sure the special type picker has a selected type's icon on the left side (it should be "ellipsis" for "No special type")
6. Change a special type for some column and make sure both table column header and type picker icons are changed accordingly

### Demo

**Before**
![semantic-type-icons-before](https://user-images.githubusercontent.com/17258145/149557674-9f825d58-bc24-4f79-ab9a-5b37cfb68224.png)

**After**
![semantic-type-icons-after](https://user-images.githubusercontent.com/17258145/149558248-610d1943-62a5-48cf-8efc-4a8578d48215.png)
